submissions/AndriiBabushko/html-css_popup/styles/css/style.css

```diff
@@ -0,0 +1,312 @@
+@import url("https://fonts.googleapis.com/css2?family=Space+Mono&display=swap");
```

Comment:
Your PR should contain files only for review. No need to add compiled files to your PR. remove them.

Reply:
✓

Reply:
Sorry, I didn't understand you right away. I added only essential files for PR.

Reply:
Гsually we do not add already compiled files, because there is no need for this since the project builds on the server as soon as we launch our changes.